frontend/src/views/styles/modules/_m-journey-embed.scss

```diff
@@ -3,7 +3,8 @@
     position: absolute;
     top: 0;
     bottom: 0;
-    left: 0;
+    // ! we are assuming here the size of the left sidebar will be always 450px
+    left: 450px;
```

Comment:
You don't need to make any assumption. You could use a flex container. I haven't fully tested this but here's the start:

- Add the following rules to `.m-journey--embed--light`:
```css
display: flex;
align-items: stretch;
justify-content: space-between;
```
- Add the following rules to `.m-journey--embed--light > .side-bar`:
```css
flex-basis: 320px;
position: relative;
flex-grow: 1;
```
- Add the following rules to `.m-journey--embed--light > .embebed-map`:
```css
flex-grow: 1;
```